### PR TITLE
Feature/date

### PR DESCRIPTION
**Pull request by saliveja**
Add date class to cgk module.

### CODE REVIEW

File: date.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    def historical_range(self):
+        """Historical asset price for date range"""
+        asset_data_range = f"https://api.coingecko.com/api/v3/coins/{id}/market_chart/range?vs_currency={vs}&from={self.time_to}&to={self.time_to}&precision=full"
+        r = requests.get(asset_data_range)
+        return asset_data_range
+    
+if __name__ == '__main__':
+    cgkdate = CgkDate()
+    cgkdate.time_from()
+
```
Comment (serinko) [inline date.py]: I guess this won't be needed as it will be implemented as a module. Seems like this is a relict from testing purposes.

**Review by serinko — CHANGES_REQUESTED** [review]
Thank you, it looks great, left one request.

File: date.py (hunk 1 above)
Comment (saliveja) [reply date.py]: removed

**Review by serinko — APPROVED** [review]

**Comment by serinko** [comment]
LGTM!

**Pull request merged**